perfkitbenchmarker/vm_util.py

```diff
@@ -117,6 +117,11 @@ def GenTempDir():
   temp_dir.CreateTemporaryDirectories()
 
 
+def FilePresent(file_path):
```

Comment:
Just lack of awareness. I have removed this method and defaulted to using the FileResourceLoader